flopy/mf6/mfpackage.py

```diff
@@ -1361,8 +1402,32 @@ def _find_repeating_datasets(self):
                 repeating_datasets.append(dataset)
         return repeating_datasets
 
+    def _prepare_external(self, fd, file_name, binary=False):
+        fd_main = fd
+        fd_path = self._simulation_data.mfpath.get_model_path(self.path[0])
+        # resolve full file and folder path
+        fd_file_path = os.path.join(fd_path, file_name)
+        fd_folder_path = os.path.split(fd_file_path)[0]
+        if fd_folder_path != "":
+            if not os.path.exists(fd_folder_path):
+                # create new external data folder
+                os.makedirs(fd_folder_path)
+        return fd_main, fd_file_path
+
     def _write_block(self, fd, block_header, ext_file_action):
         transient_key = None
+        basic_list = False
+        dataset_one = list(self.datasets.values())[0]
+        if isinstance(dataset_one, mfdataplist.MFPandasList) or isinstance(
+            dataset_one, mfdataplist.MFPandasTransientList
+        ):
```

Comment:
This could be simplified to `isinstance(dataset_one, (mfdataplist.MFPandasList, mfdataplist.MFPandasTransientList))`

Reply:
Simplified